runfolder/services.py

```diff
@@ -164,8 +164,20 @@ def get_runfolder_state(self, runfolder):
         return state
 
     @staticmethod
+    def validate_state(state):
+        """Raises InvalidRunfolderState if the state is not known"""
+        if state not in [RunfolderInfo.STATE_NONE,
```

Comment:
I don't know exactly how to do it, but it would be nice if this did not need to be enumerated here. I thinking that if one adds a new type of state it would be nice if that was automatically valid.